modules/videostab/include/opencv2/videostab/global_motion.hpp

```diff
@@ -263,6 +274,8 @@ class CV_EXPORTS KeypointBasedMotionEstimatorGpu : public ImageMotionEstimatorBa
     void setOutlierRejector(Ptr<IOutlierRejector> val) { outlierRejector_ = val; }
     Ptr<IOutlierRejector> outlierRejector() const { return outlierRejector_; }
 
+     virtual void setFrameMask(Mat val) CV_OVERRIDE { mask_ = val; }
```

Comment:
```
mask.copyTo(mask_);
```
or
```
mask_ = mask.getMat();
```

The second requires responsibility of caller to avoid "used" masks modification.
The first way is slower but more reliable.